openy_af4_vue_app/dist/activity_finder_4.umd.min.js

```diff
@@ -1,2 +1,2 @@
lteredDaysTimes:function(){if(!this.firstStep)return this.daysTimes;var t={};for(var e in this.daysTimes)this.optionsCount(e)>0&&(t[e]=this.daysTimes[e]);return t},filtersSelected:function(){return this.value.length>=1},firstItemWithOptions:function(){for(var t in this.days)if(this.optionsCount(t)>0)return t;return 0}},watch:{value:function(){this.selectedDaysTimes=this.value}},methods:{onChange:function(t,e){this.trackEvent("selectDaysTimes","Click on day "+t.search_value+" and time "+e.label,e.value),this.$emit("input",this.selectedDaysTimes)},onSkip:function(){this.trackEvent("skip","Click on selectDaysTimes"),this.$emit("input",[]),this.$emit("nextStep")},onNext:function(){this.trackEvent("next","Click on selectDaysTimes"),this.$emit("nextStep")},facetCount:function(t){var e=this.facets.find((function(e){return e.filter===t}));return e&&e.count?e.count:0},isDisabled:function(t){return 0===this.facetCount(t)},subFiltersCount:function(t){var e=this,i=0;return this.value.forEach((function(s){e.daysTimes[t].value.find((function(t){return String(t.value)===String(s)}))&&i++})),i},optionsCount:function(t){var e=0;for(var i in this.daysTimes[t].value)e+=this.facetCount(this.daysTimes[t].value[i].value);return e}}}),pt=ft,vt=O(pt,ut,dt,!1,null,null,null),ht=vt.exports,mt=function(){var t=this,e=t.$createElement,i=t._self._c||e;return i("div",{staticClass:"select-locations-component"},[i("Step",{attrs:{"skip-label":t._f("t")("All locations (Skip)"),"filters-selected":t.filtersSelected},on:{skip:t.onSkip,next:t.onNext},scopedSlots:t._u([{key:"title",fn:function(){return[t._v(" "+t._s(t._f("t")("Do you have any location preferences?"))+" ")]},proxy:!0},{key:"default",fn:function(e){var s=e.handleSticky;return t._l(t.filteredLocations,(function(e,n){return i("Fieldset",{key:n,attrs:{label:e.label,"collapse-id":"accordion-"+n,counter:t.subFiltersCount(n),"counter-options":t.optionsCount(n),accordion:"accordion-locations",collapsed:!0,"handle-sticky":s}},[i("div",{staticClass:"options"},[i("div",{staticClass:"row"},t._l(e.value,(function(e){return i("div",{key:e.value,staticClass:"option check col-12 col-xs-12 col-sm-4"},[i("input",{directives:[{name:"model",rawName:"v-model",value:t.selectedLocations,expression:"selectedLocations"}],attrs:{id:e.value,type:"checkbox",disabled:t.isDisabled(e.value)},domProps:{value:e.value,checked:Array.isArray(t.selectedLocations)?t._i(t.selectedLocations,e.value)>-1:t.selectedLocations},on:{change:[function(i){var s=t.selectedLocations,n=i.target,a=!!n.checked;if(Array.isArray(s)){var r=e.value,o=t._i(s,r);n.checked?o<0&&(t.selectedLocations=s.concat([r])):o>-1&&(t.selectedLocations=s.slice(0,o).concat(s.slice(o+1)))}else t.selectedLocations=a},function(i){return t.onChange(e)}]}}),i("label",{attrs:{for:e.value,role:"button"}},[i("span",[i("span",{staticClass:"title"},[t._v(t._s(e.label))]),i("span",{staticClass:"results-count"},[t._v(" "+t._s(t._f("formatPlural")(t.facetCount(e.value),"1 Result","@count Results"))+" ")])
```

Comment:
Why did the compiled JS changed? your changes should result in compiled CSS change only...